README.md

```diff
@@ -99,6 +99,28 @@ your local docker devstack by following these steps:
 Your devstack Studio should now be using the production studio-frontend files
 built by your local checkout.
 
+## Testing in studio-frontend
+
+### Where do test files go?
+If you are developing a component and you are adding new `js` or `jsx` files,
```

Comment:
Nit: Could you specify that this refers to test files, i.e. "and you are adding new `js` or `jsx` **test** files"?